ARM-wvd-templates/HCI/FastTrack/AddHciVirtualMachinesFastTrackTemplate.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
@shanberg1 Can we change the folder names and files names? Instead of 'FastTrack' keyword, we should call it 'GA' or something.

Reply:
+1. I don't think "fast track" is the best identifier for this as it doesn't carry much meaning to customers. I would also tend to avoid labeling them based on release status since that will change over time.

My first thought would be to differentiate them by their respective resource providers, i.e. these are `HybridCompute` and the others are `AzureStackHci`, but that's confusing because they're both for use with Azure Stack HCI.

Maybe these are the `HybridCompute` templates and the others are `AzureArc` templates? That gets to the big difference, which is the other templates result in Azure Arc VMs and these do not.

Reply:
Agree, let's go with 'HybridCompute' templates.
@shanberg1 @rick-man